.github/codecov.yml

```diff
@@ -0,0 +1,9 @@
+# Only post coverage comments and not pass/fail reports in the CI checks
```

Comment:
Do you think we need this at all given that we're not really acting on the feedback?

Reply:
I like the coverage percentages, shows the lib has good test coverage

Reply:
I use the feedback when I add new features

Reply:
> I like the coverage percentages, shows the lib has good test coverage

👍 

> I use the feedback when I add new features

When you put together a contribution guide for this repo, make sure this is clear.

As a n00b, I'd probably think `this tool I've never seen or used is giving me a ⚠️. Do I ignore it or keep going?`